docs/aurora/data-science/inference/setup_ray_cluster.sh

```diff
@@ -0,0 +1,148 @@
+
```

Comment:
I would move scripts out of this repo; you can add it to https://github.com/argonne-lcf/GettingStarted/tree/master/DataScience
which is included as a Git submodule with this repo